src/main/java/com/bulkbuy/enterprise/dto/Customer.java

```diff
@@ -0,0 +1,11 @@
+package com.bulkbuy.enterprise.dto;
+
+
+import lombok.Data;
+
+public @Data
+class Customer {
```

Comment:
added customer object, this way when customer places multiple orders their information won't need to be duplicated.
https://www.baeldung.com/java-clean-code

Reply:
Is this used anywhere?  If not, it's incomplete functionality, which is technical debt.